indico/modules/events/surveys/models/surveys.py

```diff
@@ -196,3 +229,21 @@ def close(self):
         if self.state not in (SurveyState.active_and_clean, SurveyState.active_and_answered):
             raise IndicoError("Survey can't be closed")
         self.end_dt = now_utc()
+
+    def send_start_notification(self):
+        if not self.notifications_enabled or self.start_notification_sent:
+            return
+        addresses = self.start_notification_emails
+        if self.include_participants_registrants:
+            addresses += self.event.getRegistrantsList() + self.event.getParticipation().getParticipantsList()
+
+        template_module = get_template_module('events/surveys/emails/start_notification_email.txt', survey=self)
+        send_email(make_email(bcc_list=set(addresses), template=template_module))
+        self.start_notification_sent = True
+
+    def send_notification_on_new_submission(self, submission):
```

Comment:
way too verbose method name. also, where's the docstring? ;)

I'd go for `send_submission_notification`.